docs/analytics.md

```diff
@@ -0,0 +1,56 @@
+# Analytics
+
+New Relic provides tools for application monitoring and web analytics. New Relic's [Application Performance Monitoring (APM)](https://docs.newrelic.com/docs/apm/) can monitor and measure application performance and activity on the backend. Their [Browser Agent](https://docs.newrelic.com/docs/browser/) can do the same for the frontend of the application.
+
+We can [create custom dashboards](https://docs.newrelic.com/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/) in New Relic that visualize the various metrics we collect, and combine metrics from the backend and frontend.
+
+## Metrics for Smart Hub
+
+### Version 1, 4/16/21
+
+#### Operational
+
+- % uptime
+
+- Average Response Time
+
+- Request rates
+
+- Error rates
+
+- Application/CPU usage
+
+#### Usage (by region and cumulative)
+
+- \# of registered users (in admin tool or New Relic?)
+
+- \# of closed user accounts (proxy for regional TTA turnover? Also in DOC
+SmartSheets)
+
+- \# of unique visitors
+
+- \# of visits per unique user
+
+- \# of visits by role (specialist, manager, federal staff by region)
+
+- Avg time on site by user
+
+#### Tasks (by region and cumulative)
+
+- Avg time spent on each AR
+
+- \# of visits per AR (from draft AR through approved AR)
+
+- \# of “cut & paste” on each AR (Proxy for pasting content from
+different applications or systems)
+
+- \# of “request revisions” for each AR (from manager to creator)
+
+- \# of “print” ARs (distribution to grantees or team members who want it
+outside the Hub)
+
+- \# of URL “share” ARs (distribution from TTA to other members with Hub
```

Comment:
I'm unclear how we would track this and the print metric, since the mechanism is copying the URL out of the browser URL bar or otherwise using browser/OS built-in actions.

Reply:
An `onbeforeprint` could work for tracking printing, so that one should be possible. 

I'm not sure that tracking URLs copied out of the browser bar is possible, so we'd likely need to add some data to the URLs themselves, e.g. `?referrer=userid`, if we wanted to try tracking that way.

Reply:
> An `onbeforeprint` could work for tracking printing

Ah, cool. I didn't know that was a hook